uber/templates/accounts/index.html

```diff
@@ -22,13 +28,11 @@ <h3 class="panel-title">New Account</h3>
         </select>
       </div>
       <div class="form-group">
-        <div class="input-group input-group-sm">
         <label for="password" class="sr-only">Password</label>
-        <input type="password" class="form-control" id="password" name="password" placeholder="Password"
-           {% if not c.AT_OR_POST_CON %}disabled{% endif %}>
+        <div class="input-group">
+          <input type="password" class="form-control" id="password" name="password" placeholder="Password" {% if not c.AT_OR_POST_CON %}disabled{% endif %}>
            {% if not c.AT_OR_POST_CON %}
-               <span class="input-group-addon glyphicon glyphicon-ban-circle">
-               </span>
+           <span class="input-group-addon"><span class="glyphicon glyphicon-ban-circle"></span></span>
```

Comment:
Fixes the styling of the change password fields. Not related to this PR, but sweet-christmas was it annoying me!